server/routes/support/handlers/probationTeam.ts

```diff
@@ -21,16 +22,22 @@ export default class ProbationTeamRoutes {
       view === 'prison'
         ? await this.comCaseloadService.getTeamCreateCaseload(user, [teamCode])
         : await this.comCaseloadService.getTeamVaryCaseload(user, [teamCode])
-    ).map(comCase => {
-      const releaseDate = comCase.releaseDate ? format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy') : 'not found'
-      return {
-        ...comCase,
-        releaseDate,
-        hardStopDate: comCase.hardStopDate && format(parseCvlDate(comCase.hardStopDate), 'dd/MM/yyyy'),
-        hardStopWarningDate:
-          comCase.hardStopWarningDate && format(parseCvlDate(comCase.hardStopWarningDate), 'dd/MM/yyyy'),
-      }
-    })
+    )
+      .map(comCase => {
+        const releaseDate = comCase.releaseDate ? format(parseCvlDate(comCase.releaseDate), 'dd MMM yyyy') : 'not found'
+        return {
+          ...comCase,
+          releaseDate,
+          licenceStatus: comCase.isReviewNeeded ? LicenceStatus.REVIEW_NEEDED : comCase.licenceStatus,
+          sortDate: comCase.releaseDate && parseCvlDate(comCase.releaseDate),
+          hardStopDate: comCase.hardStopDate && format(parseCvlDate(comCase.hardStopDate), 'dd/MM/yyyy'),
+          hardStopWarningDate:
+            comCase.hardStopWarningDate && format(parseCvlDate(comCase.hardStopWarningDate), 'dd/MM/yyyy'),
+        }
+      })
+      .sort((a, b) => {
+        return (a.sortDate?.getTime() || 0) - (b.sortDate?.getTime() || 0)
```

Comment:
**thought (non-blocking):** I don't think it matters with the number of cases we're dealing with, but I wonder if there's an efficiency difference between creating a `releaseDateString` that is the formatted `releaseDate`, then comparing `releaseDate` directly here.

Reply:
quite possibly, I guess it would be worth testing if it started to get slow.